docs/embedding/modular-embedding.md

```diff
@@ -7,105 +7,104 @@ redirect_from:
 
-### 3. Customize your embed
+##  Pick what you want to embed
 
-The exact customization options you see will depend on what type of entity you're embedding. You'll see a live preview of how the embed will look with your chosen options. Check out [Customizing embeds](#customizing-embeds) for more details on customization options.
+If you haven't already picked, now's the time.
 
-![Customizing embeds](./images/embed-flow-options.png)
+Click **Next** to customize your embed.
 
-You'll also be able to pick brand, text, and background color used for all your embeds. To configure other colors (e.g. secondary colors, query builder colors etc), as well as font, you can specify a [theme](#theming) in your embed code snippet.
+## Customize your embed
 
-All the customization options you select in this interactive flow will be reflected in the parameter values in the embed code, so you'll be able to adjust them later by editing the embed snippet.
+The exact customization options you see will depend on what type of entity you're embedding. You'll see a live preview of the embed with your customizations.
 
-Once you're done customizing your embed, click "Next".
+![Embed flow options for AI chat](./images/embed-flow-options.png)
 
-### 4. Select authentication method
+You'll also be able to pick colors for:
 
-You'll get a choice between "Existing Metabase session" and "Single sign-on (SSO)".
+- Brand
+- Text
+- Background
 
-- If you select **Existing Metabase session**, you'll be able to preview your embeds as the user you're currently logged into Metabase, and only in the same browser as your current session. Not all browsers are supported - we recommend using Google Chrome. To test out embedding in other contexts, you can use [API keys](#use-api-keys-to-test-embeds) instead. For production usage, use [SSO](#set-up-sso).
+But you customize a lot more. To configure other colors (like secondary colors, query builder colors, and more), as well as font, you can specify a `theme` in your embed code snippet.
 
-- If you set up JWT in your Metabase instance, you'll be able to select **Single sign-on (SSO)**, see [Set up SSO](#set-up-sso).
+All the customization options you select in this interactive flow will be reflected in code snippet, which you can edit as you see fit.
 
-### 5. Add the embedding script into your app
-
-Metabase will generate a code snippet that you can copy and paste into your app, see [Embed code snippets](#embed-code-snippets) for an example. You can later modify this code snippet to specify additional appearance options or change the behavior of some components.
-
-Add the code snippet into your app, and refresh the page.
+For example, this code defines the font, color, and size for text, background colors, and colors for filters and summaries:
```

Comment:
at this point in the wizard, they haven't seen a snippet yet. Are we walking them through the wizard or giving them general guidance?